app/models/katello/glue/pulp/repo.rb

```diff
@@ -70,166 +29,32 @@ def initialize(attrs = nil)
       end
 
       def srpm_count
-        pulp_repo_facts['content_unit_counts']['srpm']
+        # TODO: Pulp 3 replacement?
```

Comment:
No need to replace, you can get rid of it entirely.

Reply:
The main reason I didn't do this yet was that I then also needed to make UI adjustments and I wasn't ready for that yet.

Reply:
You shouldn't have to worry about UI changes regarding any of the counts in this file.  The `srpm_counts` mentioned in the UI refer to a content view versions `content_counts`.  The code where we dynamically load these counts (which is why it's not grep-able) is here: https://github.com/Katello/katello/blob/master/app/models/katello/content_view_version.rb#L333

The `content_counts` get passed to the API through the method `content_counts_map`: https://github.com/Katello/katello/blob/master/app/views/katello/api/v2/content_view_versions/base.json.rabl#L13